scripts/join_pane.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+source "$CURRENT_DIR/helpers.sh"
+
+join_pane() {
+	tmux command-prompt -p "join pane to:" "join-pane -t '%%'"
```

Comment:
How do you specify the target pane? Via pane id? Isn't that complex?

Reply:
i tried passing in `#{pane_id}` originally but it never seemed to do the right thing, 
the default (with not specifying `-s` at all) was the version that behaved as expected :confused: